src/components/molecules/bar-chart.tsx

```diff
@@ -277,6 +277,7 @@ const styles = StyleSheet.create({
   legendLabel: {
     ...text.small,
     textAlign: 'left',
-    marginLeft: 8
+    marginLeft: 8,
+    lineHeight: 15
```

Comment:
Is there any reason that this lineHeight isn't coming in with the spread of `text.small`?